### PR TITLE
Use expressions for property getters where possible

### PR DESCRIPTION
**Pull request by hughbe**
Find: `(\{[\n\r\s]*get)[\n\r\s]*\{[\n\r\s]*return[\n\r\s]*(.*)[\n\r\s]*\}([\n\r\s]*set)`
Replace: `$1 => $2$3`

### CODE REVIEW

**Comment by RussKie** [comment]
I'm not a big fan of this change because it introduces stylistic inconsistencies in a number of instances
```
public int MyProperty
{
    get => _myProperty;
    set 
    {
        // a number of lines
    }
}
```

File: src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCellCollection.cs [hunk 1]
```diff
@@ -94,7 +94,7 @@ bool IList.IsReadOnly
         /// <internalonly/>
         object IList.this[int index]
         {
-            get { return this[index]; }
+            get => this[index];
```
Comment (danmosemsft) [inline src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCellCollection.cs]: what about indexer setters?

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/CurrentDPILabel.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ public CurrentDPILabel()
         [DefaultValue(false)]
         public override bool AutoSize 
         {
-            get { return base.AutoSize; }
+            get => base.AutoSize; 
             set { base.AutoSize = value; }
```
Comment (danmosemsft) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/CurrentDPILabel.cs]: setters?

**Comment by danmosemsft** [comment]
@zsd4yr should decide what style his team prefers. My suggestion is to use expression bodied (for properties, event handlers and indexers) iff getter and setter are each one line - and then do it for both. I think that is generally what we do in CoreFX - even if it is not written down.

That would mean more regexing to do the setters, then a bunch of time selectively reverting the diffs.

**Pull request closed without merging**